src/app/App.tsx

```diff
@@ -95,7 +95,10 @@ export const App = (): JSX.Element => {
     configErrors === VaultErrors.CONNECTION_FAILED
   ) {
     content = (
-      <Section header={<SectionHeader title="Failed to connect" />}>
+      <Section
+        header={<SectionHeader title="Failed to connect" />}
+        hideNotifications={true}
```

Comment:
We generally try to conform with `is/has` prefix for boolean props convention if possible. For example:

```suggestion
        isNotificationListHidden
```

Reply:
Ahh okay! I'll change that